### DISCUSSION TITLE
Node temperature

### DISCUSSION THREAD

**Discussion by ulinieto** [dougiteixeira/proxmoxve#346] [Feature request] [answered]
Hello, there's a way to get the computer's temperature where proxmox is installed?? 

Thx in advance for your great work.

**Comment by moshoari** [comment]
I’m also looking for this feature. It would be really helpful to have CPU temperature monitoring per node. Please go and add a comment to the link below.
https://github.com/dougiteixeira/proxmoxve/discussions/285#discussion-6416316

**Comment by mZ738** [comment, marked as answer]
Hardware readings are in since 5.0.0: temperatures, fan speeds, voltages and power from `lm-sensors`, through [PVE-mods](https://github.com/Meliox/PVE-mods) on the node — the integration reads them from the node status the mod extends, no extra API calls. Both the current `node_info` package and the legacy script are recognised. How to set it up, which chips are classified how, and why readings come and go with the mod's on-demand collector: [Hardware sensors](https://github.com/dougiteixeira/proxmoxve/blob/main/docs/hardware-sensors.md). 5.1.0 ([release notes](https://github.com/dougiteixeira/proxmoxve/releases/tag/5.1.0)) is out today; I am closing this as done — please open a new thread if a chip of yours is not picked up.
